### PR TITLE
Accessibility tests

### LINKED ISSUE #112: Automate Storybook Accessibility Testing
### What is this for?

website

### Feature

As a developer, I want to 

### Ideal solution or implementation

Use these as a reference:
https://www.youtube.com/watch?v=rNLL0SICr9w
https://www.deque.com/axe/
https://storybook.js.org/blog/automate-accessibility-tests-with-storybook/

### Alternative solutions or implementations

_No response_

### Other context

_No response_

### PR DESCRIPTION
**Pull request by JoltCode**
Closes #112

### CODE REVIEW

**Comment by JoltCode** [comment]
Closes #112

**Comment by JoltCode** [comment]
Also @RhysSullivan seems I've accumulated some conflictions in my files changed...

File: packages/ui/package.json [hunk 1]
```diff
@@ -7,8 +7,11 @@
   "scripts": {
     "lint": "next lint && tsc --noEmit",
     "lint:fix": "next lint --fix",
-    "dev": "storybook dev -p 6006 --quiet --no-open",
-    "test-storybook": "test-storybook",
+    "dev": "cross-env THEME=\"both\" storybook dev -p 6006 --quiet --no-open",
+    "dev:dark": "cross-env THEME=\"dark\" storybook dev -p 6006 --quiet --no-open",
```
[the remark this answers is not in the document]
Comment (JoltCode) [reply packages/ui/package.json:11]: Okay, it appeared there was some weird behaviour though with the tests, so for the moment when running the tests it seems we need to use the environment we're testing for just to be safe (needs further investigation)

[the remark this answers is not in the document]
Comment (JoltCode) [reply packages/ui/package.json:11]: So for dark theme -> dev:dark + test-storybook:dark

File: packages/ui/src/components/analytics/Chart.stories.tsx [hunk 2]
```diff
@@ -4,6 +4,14 @@ import type { Meta, StoryFn } from "@storybook/react";
 import { Chart, ChartProps } from "./Chart";
 export default {
   component: Chart,
+  // For the moment, we need to disable a11y checks for the charts - failing a11y checks
```
[the remark this answers is not in the document]
Comment (JoltCode) [reply packages/ui/src/components/analytics/Chart.stories.tsx:7]: #120

File: packages/ui/src/components/Message.stories.tsx [hunk 3]
```diff
@@ -81,7 +81,7 @@ WithCode.args = {
   message: {
     ...default_message.message,
     content: `
-    \`\`\`typescript
+    \`\`\`html
```
[the remark this answers is not in the document]
Comment (JoltCode) [reply packages/ui/src/components/Message.stories.tsx]: Done [d5c3b68](https://github.com/AnswerOverflow/AnswerOverflow/pull/118/commits/d5c3b680f688cb1280362274fc26e4c805464771)

File: packages/ui/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (JoltCode) [reply packages/ui/package.json:11]: Yup, as I said it's a tad unstable as well.... :frowning_face:

**Review by RhysSullivan — APPROVED** [review]
LGTM!

**Pull request merged**